ibm/mas_devops/roles/ocp_provision/templates/fyre/quick_burn.json.j2

```diff
@@ -17,5 +17,6 @@
             "http-keep-alive": "10s",
             "check": "10s"
         }
-    }
+    },
+    "fips":"{{ ocp_fips }}"
```

Comment:
- Use the new bool to convert from true/false to "yes"/"no" string specific for fyre.
- Should we not also add the same to `product_group.json.j2` as well?